diode-server/netbox/ipam_test.go

```diff
@@ -0,0 +1,86 @@
+package netbox
```

Comment:
question. cant it be ipam_test / netbox_test (external package)?

Reply:
good catch